app/controllers/api/v1/courses_controller.rb

```diff
@@ -18,21 +19,27 @@ def index
                                  :subjects,
                                  site_statuses: [:site])
                        .changed_since(changed_since)
-                       .by_recruitment_cycle(recruitment_year)
+                       .by_recruitment_cycle(@recruitment_cycle.year)
```

Comment:
If you look at the implementation for `by_recruitment_cycle`:

```ruby
scope :by_recruitment_cycle, ->(recruitment_year) {
  joins(provider: :recruitment_cycle).merge(RecruitmentCycle.where(year: recruitment_year))
}
```

It seems a bit odd to be instantiate a recruitment cycle in the controller action using the year, then passing the year value just to instantiate another recruitment cycle object using the year.

Not a big deal as this only happens once per index, but maybe this would be better expressed like this:

```ruby
@courses = @recruitment_cycle
             .courses
             .includes(:provider
                       :site_statuses,
                       :accrediting_provider,
                       :subjects,
                       site_statuses: [:site])
             .changed_since(changed_since)
             .limit(per_page)
```

But not a blocker.

Reply:
Done.